layouts/index.ts

```diff
@@ -1,4 +1,4 @@
 export { AuthLayout } from "./AuthLayout";
 export { AuthPagesLayout } from "./AuthPagesLayout";
 export { BaseLayout } from "./BaseLayout";
-export { DashBoardPageLayout } from "./DashBoardPageLayout";
+export { DashBoardLayout } from "./DashBoardPageLayout";
```

Comment:
name the component with the same name as the file name. you can clearly see that is the convention used